actors/common/pom.xml

```diff
@@ -44,12 +44,12 @@
 		<dependency>
 			<groupId>com.typesafe.akka</groupId>
 			<artifactId>akka-actor_2.11</artifactId>
-			<version>2.5.16</version>
+			<version>2.5.19</version>
```

Comment:
put it in one place{{<version>2.5.19</version>}}
applicable for all repeated version

Reply:
Done